### DISCUSSION TITLE
Slightly confusing translation in AutoWarmth

### DISCUSSION THREAD

**Discussion by Frenzie** [koreader/koreader#9990] [Q&A]
@zwim I was wondering what this does exactly? I was doing some translating today and ran into the rather cryptic string `" %1 "`, which seems to be replaced by a translation of and.
https://github.com/koreader/koreader/blob/a8b333e4f91bfcb2c7b0c569aad4c0f04e6aa8ed/plugins/autowarmth.koplugin/main.lua#L1045

**Comment by ghost** [comment]
The good news is, that our translators are not bothered with spaces before and after `and`, the bad news, they have to translate `" %1 "`. 
The other good news is, that this is easily fixable with:
```
self.control_warmth and self.control_nightmode and T(" %1 ", _("and")) or "", 
```
